src/client/config/constants.js

```diff
@@ -54,3 +54,10 @@ export const PICTURE_QUALITIES = {
 };
 export const DEFAULT_PICTURE_QUALITY = PICTURE_QUALITIES.LARGE;
 export const MAX_COLLECTION_NAME_LENGTH = 100;
+
+// Cache Keys
+export const COLLECTIONS = 'collections';
```

Comment:
Sorry to make you change it once again... What do you think of adding something like `KEY` or something related to react-query? I feel like that `COLLECTIONS` could be anything.